tests/ui/pages/desktop/home.py

```diff
@@ -39,3 +90,25 @@ def users(self):
                 """Number of users that have downloaded the extension"""
                 users_str = self.find_element(*self._users_locator).text
                 return int(users_str.split()[0].replace(',', ''))
+
+    class FeaturedThemes(Region):
+
+        _featured_themes_see_all_link = (
+            By.CSS_SELECTOR, "#featured-themes h2 a")
+        _featured_themes_title_locator = (
+            By.CSS_SELECTOR, "#featured-themes h2")
+        _featured_themes_items_locator = (
+            By.CSS_SELECTOR, "#featured-themes li")
+
+        @property
+        def _wait_for_themes_page(self):
+            self.wait.until(lambda _: 'themes' in self.selenium.current_url)
+
+        def click_see_all_link(self):
+            self.selenium.find_element(
+                *self._featured_themes_see_all_link).click()
+
+        @property
+        def count(self):
+            return len(self.selenium.find_elements(
+                *self._featured_themes_items_locator))
```

Comment:
Replace this with a property that returns the themes as regions, which can then be used with `len()` in the tests.